libs/cli-kontrol-api/specs/api.yaml

```diff
@@ -321,19 +321,30 @@ components:
           description: Label for the node.
         type:
           type: string
-          enum: [gateway, service, service-version, redis]
+          enum: [gateway, service, external]
           description: Type of the node
-        parent:
-          type: string
-          description: Parent node
         versions:
           type: array
           description: Node versions
           items:
-            type: string
+            $ref: "#/components/schemas/NodeVersion"
       required:
         - id
         - type
+        - label
+
+    NodeVersion:
+      type: object
+      properties:
+        flow-id:
+          type: string
+        image-tag:
+          type: string
+        is-baseline:
+          type: boolean
```

Comment:
@laurentluce would it be possible to use a key naming scheme that is compatible with JS object notation? For example snake_case or camelCase? kebab-case cannot be used in a javascript object without indexing using strings, which is awkward. 

```js
version.flow_id;     // valid, preferred
version.flowId;      // valid, preferred
version.flow-id;     // not valid javascript
version["flow-id"];  // forced to fall back to string indexing for keys with a hypen character, not preferred
```

Reply:
Changed those to camelCase.  I will make the other attributes in that file consistent in a separate PR.